generators/generator-base-private.js

```diff
@@ -1572,4 +1572,34 @@ module.exports = class extends Generator {
             );
         }
     }
+
+    processJavaEntityImports(fields, relationships) {
```

Comment:
I'm not really convinced that improves the maintainability of the code, since this is really tight with the template.
At the very least, this function should be in the `entity-server` generator, since it's the only was using it.

Reply:
I moved it to entity sub gen